### DISCUSSION TITLE
How to search and download GEDI L2A dataset interactively

### DISCUSSION THREAD

**Discussion by oluwaseyigezekiel** [opengeos/leafmap#756] [Q&A] [answered]
https://leafmap.org/notebooks/85_gedi/ contains the source codes to search and download GEDI **L4A** dataset interactively.
Please, could you provide the codes to search and download GEDI **L2A** dataset?

**Comment by giswqs** [comment, marked as answer]
See https://leafmap.org/notebooks/88_nasa_earth_data/
![image](https://github.com/opengeos/leafmap/assets/5016453/a4f6b4d3-5e11-407a-a4b3-2cc6f7cb41cf)

**Reply by oluwaseyigezekiel** [reply]
Perfect! 
Thank you!
